website_sale_wishlist/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
He meant like this https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3